src/js/knowledgeGraph.js

```
@@ -0,0 +1,125 @@
+
+let graphNetwork = null;
+let graphInitialized = false;
+
+function buildGraphData() {
+  const nodes = [];
+  const edges = [];
+  const techNodes = {};
+  let nodeId = 1;
+
+  // Add org nodes
+  ORGS.forEach((org, idx) => {
```

ESLint

[error] 'ORGS' is not defined.

(no-undef)